common/utils/utils.go

```diff
@@ -611,12 +611,12 @@ func CompareSchema(conv1, conv2 *internal.Conv) error {
 			for _, spannerTableIndex := range spannerTable.Indexes {
 				if sessionTableIndex.Name == spannerTableIndex.Name {
```

Comment:
We should move to using ids instead of comparing with names, then 1 loop can be avoided

Reply:
Method updated. Please review it.